src/app/shared/comcol-forms/comcol-form/comcol-form.component.html

```diff
@@ -12,11 +12,13 @@
           <div class="col-4 d-inline-block">
             <div *ngIf="logo" class="btn-group btn-group-sm float-right" role="group">
               <button *ngIf="!markLogoForDeletion" type="button" class="btn btn-danger"
+                      [attr.aria-label]="(type.value + '.edit.logo.delete.title' | translate)"
                       title="{{type.value + '.edit.logo.delete.title' | translate}}"
                       (click)="deleteLogo()">
                 <i class="fas fa-trash" aria-hidden="true"></i>
               </button>
               <button *ngIf="markLogoForDeletion" type="button" class="btn btn-warning"
+                      [attr.aria-label]="(type.value + '.edit.logo.delete-undo.title' | translate)"
```

Comment:
Same issue occurs here.  Again, you already have a `title` on the next line, but neither the `title` nor the `aria-label` appear in the final source.